java/src/apps/startup/StartupModel.java

```diff
@@ -28,7 +28,7 @@
      *
      * @param name the name, an empty string, or null
      */
-    public void setName(@Nullable String name);
+    public void setName(@CheckForNull String name);
```

Comment:
Because `@CheckForNull` records that, well, checking is required before dereferencing while leaving it blank does not.  (And `@Nullable` doesn't work)